runtime/doc/eval.txt

```diff
@@ -1552,6 +1552,17 @@ v:event		Dictionary of event data for the current |autocommand|.  Valid
 					operation.
 			regtype		Type of register as returned by
 					|getregtype()|.
+			completed_item    Current selected complete item on
+					|MenuPopupChanged|, Is `{}` when no complete
```

Comment:
Should it be `v:null` when no complete?

Reply:
`{}` is consistent with how `v:completed_item` behaves, so let's remain consistent.

Reply:
That makes sense.